alexandria/core/models.py

```diff
@@ -220,6 +223,9 @@ class Variant(models.TextChoices):
         blank=True,
     )
 
+    content_vector = SearchVectorField(null=True, blank=True)
```

Comment:
You need to add an explicit index on this field, otherwise the search will not perform much better than any naive `__icontains` lookup

Reply:
Done